src/server/queries/users_dashboards.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+var pool = require('../db/index.js');
+
+module.exports = {
+  findOrCreate: function (githubId, dashboardId) {
+    // if create, initialize with set_up=0. otherwise do nothing
+    // no need to return anything (we don't think)
+  },
+  getSignatureHash: function (githubId, dashboardId) {
```

Comment:
I think we should replace this with a getOne() like all the other queries for consistency. It's more general in case we need to find one users_dashboards entry in the future as well.